README.md

```diff
@@ -41,6 +41,21 @@ See [`allTargets`](https://github.com/lgeiger/node-abi#usage) for all available
 
 For more options run `prebuild --help`. The prebuilds created are compatible with [node-pre-gyp](https://github.com/mapbox/node-pre-gyp)
 
+If you'd like to include other files with your prebuilds like additional
+`.node` files or other native libraries, you can pass a file-matching regular
+expression to the  `--collect-files-filter` parameter:
```

Comment:
you can pass a file-matching regular expression to the  `--collect-files-filter` parameter

->

you can pass a file-matching regular expression to `--collect-files-filter`:

Reply:
👍